src/viking.js

```diff
@@ -1,11 +1,70 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(thedamage) {
+    this.health -= thedamage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+
+    this.name = name;
+  }
+  receiveDamage(thedamage) {
+    this.health -= thedamage;
+    if (this.health > thedamage) {
```

Comment:
A comparação deveria ser 0 ao invés de damage, por queremos saber se após a vida ter sido reduzida pelo dano, o viking continua vivo ou não. No caso o teste passa por causa de um falso positivo

```suggestion
    if (this.health > 0) {
```